website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx

```diff
@@ -230,11 +230,15 @@ Configure [VCS providers](/terraform/cloud-docs/vcs) to use in the organization.
 Data retention policies are exclusive to Terraform Enterprise, and not available in Terraform Cloud. <a href="https://developer.hashicorp.com/terraform/enterprise">Learn more about Terraform Enterprise</a>.
 </EnterpriseAlert>
 
-By default, the **Do not auto-delete data** option is enabled for an organization. This option directs Terraform Enterprise to retain data associated with configuration and state versions, but organization owners can define configurable data retention policies that allow Terraform to _soft delete_ the backing data associated with configuration versions and state versions. Soft deleting refers to marking a data object for garbage collection so that Terraform can delete the object after a set number of days.
+There are three types of data retention policies that an organization owner has the ability to set or override: the admin default policy, do not auto-delete, and auto-delete data.
+
+If the admin default policy is selected, all other selections will be disabled by default until the admin default policy checkbox is unchecked.
+
```

Comment:
```suggestion
An organization owner can set or override the following data retention policies: 

- **Admin default policy** 
- **Do not auto-delete** 
- **Auto-delete data**

Setting the data retention policy to **Admin default policy** disables the other data retention policy settings.

```
Ah, OK. This is a little clearer than how it's documented in [PR 1040](https://github.com/hashicorp/ptfe-releases/pull/1040)